### ISSUE TITLE
Your site not availible

### ISSUE THREAD

**Issue by mAdloVe** [JMaNGOS/JMaNGOS#36]
![36](https://f.cloud.github.com/assets/914622/1143453/aebba754-1d3d-11e3-9ae9-63fc48d614e7.jpg)

**Comment by ZuSe** [comment]
madlove are you going to give jmangos a second try or why all the issues?

**Comment by mAdloVe** [comment]
Еру project never stayed, but there are problems in the management of them.
I hope this is temporary.

**Comment by ZuSe** [comment]
The website is down not reachable.
I really would like to contribute since i believe in a java based core, especially for new developers it can be a good starting point for java enterprise development. 

Are you in contact with MinimaJack?

**Comment by mAdloVe** [comment]
> The website is down not reachable.

still at least week will not be available

> I really would like to contribute since i believe in a java based core, especially for new developers it can be a good starting point for java enterprise development.

we are always happy to help in the project, but a sea change need to negotiate with all project team

> Are you in contact with MinimaJack?

yes

**Comment by ZuSe** [comment]
Yeah, the java ee migration needs to be discussed. I can only recommed it ;)

**Comment by mAdloVe** [comment]
What do you propose to run java ee 7?

**Comment by ZuSe** [comment]
Glassfish, thats the reference implementation.

**Comment by ZuSe** [comment]
madlove, check this:
https://github.com/ZuSe/AWEPServer/tree/master

I will have some fun and try how hard the migration to JavaEE7 would be

**Comment by MinimaJack** [comment]
spring-data-jpa...ehcache-spring-annotations, spring, spring, spring .... so i want to see easy "migration to JavaEE7" ))

**Comment by ZuSe** [comment]
ui, the master is back!

**Comment by mAdloVe** [comment]
but that isn't necessary now
